app/scripts/app.js

```diff
@@ -88,7 +88,21 @@ angular
       }
     };
   })
+  .factory('loggedOutInterceptor', function($q, alertService) {
+    return {
+      responseError: function (rejection) {
+        var status = rejection.status;
+        if (status === 401) {
```

Comment:
We could do this. The distinction must be made on the server and another HTTP statuscodes must be returned (i.e. 418).